scoring/general-helpers.metta

```diff
@@ -0,0 +1,112 @@
+
+
+(= (addToSpace $row $space) (add-atom $space $row))
```

Comment:
Why did you need this function? Why not use `add-atom` it self instead of this new functions?